package.json

```diff
@@ -130,31 +120,6 @@
                 }
             }
         ],
-        "menus": {
-            "commandPalette": [
-                {
-                    "command": "deviceSimulatorExpress.runSimulatorEditorButton",
-                    "when": "false"
-                }
-            ],
-            "editor/title": [
-                {
-                    "when": "editorLangId==python  && config.deviceSimulatorExpress.showOpenIconInEditorTitleMenu",
-                    "command": "deviceSimulatorExpress.openSimulator",
-                    "group": "navigation@1"
-                },
-                {
-                    "when": "editorLangId==python && config.deviceSimulatorExpress.showSimulatorIconInEditorTitleMenu",
-                    "command": "deviceSimulatorExpress.runSimulatorEditorButton",
-                    "group": "navigation@2"
-                },
-                {
-                    "when": "editorLangId==python && config.deviceSimulatorExpress.showDeviceIconInEditorTitleMenu",
-                    "command": "deviceSimulatorExpress.runDevice",
-                    "group": "navigation@3"
-                }
```

Comment:
Why are we removing these? Is the command palette going to be the only way to run the simulator?

Reply:
Yeah. The idea for now is removing these icons because adding more devices would make these icons a little less obvious as to what they would do. Also they appear in every .py file, which can get annoying.

Reply:
Thanks for clarifying! LGTM.